src/test/SeatsioChartManager.test.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Enzyme, {mount} from "enzyme";
+import Adapter from 'enzyme-adapter-react-16';
+import { SeatsioChartManager } from "../main/index";
+import Embeddable from "../main/Embeddable";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("SeatsioChartManager", () => {
+    let seatsioMock = {
+        ChartManager: class {
+            constructor(props) {
+                this.props = props;
+            }
+
+            render() {
+                return this;
+            }
+        }
+    };
+
+    Embeddable.prototype.loadSeatsio = () => {
+        return Promise.resolve(seatsioMock);
+    };
+
+    it('renders the chart manager', () => {
+        let chart = mount((
+            <SeatsioChartManager/>
+        ));
+
+        expect(chart.find('div#chart').length).toEqual(1);
+    });
+
+    it('passes parameters onto the chart manager', () => {
+        return new Promise(resolve => {
+            mount((
+                <SeatsioChartManager
+                    id="someID"
+                    className="someClassName"
+                    publicKey="aPublicKey"
```

Comment:
I see you're passing in the public key here. But shouldn't the chart manager require the secret key?

Reply:
Ah yes. Oversight from copy-pasting the tests. This will pass either way since we just check the props were passed properly, and not check if they fail to validate. Which I guess it makes sense, that’s a chartjs test, not a React one.

That said I’ve changed this to `secretKey` now to make sense 😄

Reply:
Ah, I see. Thanks 👍